packages/components/storybook/story-loader.js

```diff
@@ -0,0 +1,17 @@
+// @flow
```

Comment:
I think that this whole file should be ignored.

Reply:
Removed from commit (actually I'm not sure why it's here)

Reply:
Thanks, it was probably just some leftover.